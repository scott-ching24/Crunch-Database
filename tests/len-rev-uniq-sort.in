set a 1 1 3 3 5 5 7 7 8 6 6
len a
uniq a
get a
set b 1 7 6 7 4 9 6
sort b
get b
rev b
get b
bye
