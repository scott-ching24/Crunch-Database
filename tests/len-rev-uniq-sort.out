> ok

> 11

> ok

> [1 3 5 7 8 6]

> ok

> ok

> [1 4 6 6 7 7 9]

> ok

> [9 7 7 6 6 4 1]

> bye
